README.md

```diff
@@ -1,2 +1,112 @@
-# siddhi-plugin-vscode
-Siddhi extension for VSCode Editor
+# Siddhi Extension for Visual Studio Code
+VSCode Siddhi extension provides rich Siddhi development capabilities in VSCode.
+
+# Table of Contents
```

Comment:
No need for a table of contents, Let's follow introduction with QuickStart and how to set up the extension. No need to go on detail on how to install the extension, since we assume if they are looking for VSCode extensions, the user should know how to install it